controllers/storagecluster/storageclasses.go

```diff
@@ -231,6 +231,38 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	}
 }
 
+// newCephFilesystemStorageClassConfiguration generates configuration options for a Ceph Filesystem StorageClass.
+func newCephNetworkFilesystemStorageClassConfiguration(initData *ocsv1.StorageCluster) StorageClassConfiguration {
+	persistentVolumeReclaimDelete := corev1.PersistentVolumeReclaimDelete
+	allowVolumeExpansion := false
+	return StorageClassConfiguration{
+		storageClass: &storagev1.StorageClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: generateNameForCephNetworkFilesystemSC(initData),
+				Annotations: map[string]string{
+					"description": "Provides RWO and RWX Filesystem volumes",
+				},
+			},
+			Provisioner:   fmt.Sprintf("%s.nfs.csi.ceph.com", initData.Namespace),
+			ReclaimPolicy: &persistentVolumeReclaimDelete,
+			// AllowVolumeExpansion is set to true to enable expansion of OCS backed Volumes
+			AllowVolumeExpansion: &allowVolumeExpansion,
```

Comment:
its set to false at line 237

Reply:
> its set to false at line 237

Yup, ceph-csi does not currently allow it. 
and it receives a pointer to a bool. 

Should I change the variable name? any suggestions?

Reply:
Just Remove the comment it should be fine.

Reply:
> Just Remove the comment it should be fine.

okay, :laughing: totally missed that comment part.